docs/sources/community/design-documents/2020-02-Promtail-Push-API.md

```diff
@@ -2,8 +2,8 @@
 title: Promtail Push API
 description: Promtail Push API
 aliases: 
-- /docs/loki/latest/design-documents/
-- /docs/loki/latest/community/design-documents/
+- ../design-documents/2020-02-promtail-push-api/
+- ../community/design-documents/2020-02-promtail-push-api/
```

Comment:
This alias redirects http://localhost:3002/docs/loki/latest/community/community/design-documents/2020-02-promtail-push-api/ to this page. I don't believe that is intended behavior

```suggestion
```

Reply:
This is the intended behavior, the design docs have moved under community.

Reply:
But the URL in my comment has "community" twice.

Reply:
Why would it be doing that?

Reply:
If you follow the relative path `../community/design-documents/2020-02-promtail-push-api/` from the page `/docs/loki/latest/community/design-documents/2020-02-promtail-push-api/` you end up at `/docs/loki/latest/community/community/design-documents/2020-02-promtail-push-api/`.


For the page `/docs/loki/latest/community/design-documents/2020-02-promtail-push-api/`:
- `.` is `/docs/loki/latest/community/design-documents/`.
-`..` is `/docs/loki/latest/community/`
- `../community` is `/docs/loki/latest/community/community/design-documents/2020-02-promtail-push-api/`
- `../community/design-documents/` is `/docs/loki/latest/community/community/design-documents/`
- `../community/design-documents/2020-02-promtail-push-api/` is `/docs/loki/latest/community/community/design-documents/2020-02-promtail-push-api/`

Reply:
I think the Hugo behavior can be confusing because `.` might not be what you expect it to be.

Reply:
I'm sorry, Hugo has a lot of weird edges like this. The most valuable test is using the browser or curl to verify that the redirects are in place. 

It's hard to do a broad PR like this without having seen an example one first. If you'd like, I can open a replacement PR that adds the appropriate aliases and which can serve as a reference for the future?

Reply:
If you do the work for me then I don't learn how to do it for myself.